views/signup.jade

```diff
@@ -99,50 +109,50 @@ block content
         if useLocal
             h4 Create a new Account
 
-            form(role='form' action='/signup' onsubmit='return verifyContent();' method='post')
+            form(role="form" action="/signup" onsubmit="return verifyContent();" method="post")
                 .form-group
-                    label(for='firstname') First Name:
-                    input(type='string' id='firstname' name='firstname').form-control
+                    label(for="firstname") First Name:
+                    input(type="string" id="firstname" name="firstname").form-control
                     small
-                        span(id='errorFirstname' style='color:red')
+                        span.red(id="errorFirstname")
                 .form-group
-                    label(for='lastname') Last Name:
-                    input(type='string' id='lastname' name='lastname').form-control
+                    label(for="lastname") Last Name:
+                    input(type="string" id="lastname" name="lastname").form-control
                     small
-                        span(id='errorLastname' style='color:red')
+                        span.red(id="errorLastname")
                 .form-group
-                    label(for='email') Email address:
-                    input(type='string' id='email' name='email').form-control
+                    label(for="email") Email address:
+                    input(type="string" id="email" name="email").form-control
                     small
-                        span(id='errorEmail' style='color:red')
+                        span.red(id="errorEmail")
```

Comment:
Any specific reason for changing span(id='errorEmail' style='color:red') to span.red(id="errorEmail") ?

Reply:
There should be no inline styles, ever, unless it's injected by javascript to complete a function

Reply:
Chris, unless it is not working we should not touch pre-existing open source code. We will need to file this as PR to OS code. Minimum the changes, better the chances to get it merged.

Reply:
open source - original source code is made freely available and may be redistributed and modified why does it have to be tied back we are hosting this on our end

Reply:
Idea is to keep OS code as close as original so that it is easier for us to "upgrade" when OS code has new changes, and this is going to happen soon (authors of wicked are working to support DB). You are free to add anything to our controlled "static" files